connectors/src/connectors/slack/lib/slack_client.ts

```diff
@@ -117,7 +117,26 @@ export type SlackUserInfo = {
   image_512: string | null;
 };
 
-export async function getSlackUserInfo(
+export async function getSlackUserOrBotInfo(
+  slackClient: WebClient,
+  userId: string
+) {
+  if (userId.startsWith("B")) {
```

Comment:
Is this some king of Slack conversation?

Reply:
Assuming it is but this maybe deserves a comment :-)

Reply:
➕

Reply:
all addressed in this PR: https://github.com/dust-tt/dust/pull/5064

@flvndvd you are requested as reviewer but @spolu feel free to add yourself as a reviewer if needed.